File: Simple/Savings.cpp
#include <iostream>
#include <cmath>

int main(int argc, char const *argv[])
{
    double saving;
    std::string prodName;
    double price;

    std::cout<<"What is your monthly savings?"<<std::endl;
    std::cin>>saving;
    std::cout<<"What do you want to buy?"<<std::endl;
    std::cin>>prodName;
    std::cout<<"What is the price?"<<std::endl;
    std::cin>>price;

    double waitTime = price/saving;

    std::cout<<"You need to save for "<<std::round(waitTime)<<" months to buy "<<prodName;
    
    return 0;
}

File: Simple/Savings.java
import java.util.*;

public class Savings {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("What do you wanted to buy?");
        String whatToBuy = scan.nextLine();
        System.out.println("What is your monthly saving limit?");
        int savingMonthlyLimit = scan.nextInt();
        System.out.println("What is the price?");
        double price = scan.nextDouble();

        double waitTime = price / savingMonthlyLimit;
        System.out.println("You need to save for " + Math.round(waitTime) + " Months to buy " + whatToBuy);
    }

}
